Hazel.UnitTests/Dtls/TestDtlsHandshakeDropUnityConnection.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Hazel.Dtls;
+using System.Net;
+
+namespace Hazel.UnitTests.Dtls
+{
+    internal class TestDtlsHandshakeDropUnityConnection : DtlsUnityConnection
```

Comment:
The Connection parent class has a TestDropRate, which is useful, but I wanted to be able to specifically drop a certain event and have a more robust test. The TestDropRate makes it unclear which messages are dropped